src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(`\n`).filter(x => x.length > 4);
+
+  const object = array.reduce((prev, oneElement) => {
+    const name = oneElement.slice(0, oneElement.indexOf(':')).trim();
+    const value = oneElement.slice(
+      oneElement.indexOf(':') + 1, oneElement.length - 1).trim();
+
```

Comment:
`oneElement.indexOf(':')` -- It would be better to move it to some var and avoid double calculation(in 2 `slice`'s)

Reply:
helper -- a small function that does some simple logic